vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/resources/META-INF/resources/frontend/datepickerConnector.js

```diff
@@ -82,13 +75,6 @@ import dateFnsIsValid from 'date-fns/isValid';
                     console.warn("The locale is not supported, using default locale setting(en-US).");
                 }
 
-                let currentDate = false;
-                let inputValue = getInputValue();
-                if (typeof datepicker.i18n.parseDate === 'function' && inputValue) {
-                    /* get current date with old parsing */
-                    currentDate = datepicker.i18n.parseDate(inputValue);
-                }
```

Comment:
The `currentDate` was used in some construct below to force the date picker to rerender after changing the formatting. That seems unnecessary when replacing the whole `i18n` object as is done with the new solution.